### ISSUE TITLE
Invalid Link Anchor

### ISSUE THREAD

**Issue by doodlewind** [xcatliu/pagic#43]
We're trying to migrate [cn.history.js.org](https://cn.history.js.org) with Pagic, generally speaking it works great :) Here's the major issue related with hyperlink (since link is heavily used in our project):

**Describe the bug**
Link between pages can't be correctly anchored via direct click.

**To Reproduce**
Generate the site with following `README.md`:

``` markdown
# Hello world

<!-- Valid -->
<a href="./test.html#foo">Go to test#foo</a>

<!-- Invalid -->
<a href="./test.md#foo">Go to test#foo</a>

<!-- Not working with direct click, only works with cmd+click -->
[Go to test/#10](./test.md#10)
```

And `test.md`:

``` markdown
# Test

<a id="foo">foo</a>

## 1
## 2
## 3
## 4
## 5
## 6
## 7
## 8
## 9
## 10
```

Then just test clicking the links in README.

**Expected behavior**
All 3 links should be correctly anchored.

**Context**
- Pagic version: 0.9.1
- Deno version: 1.3.3
- Browser version (optional): 
- OS version: macOS 10.15.6

**Comment by xcatliu** [comment]
Confirmed, will be fixed in next version. Thank you for you report!

**Comment by xcatliu** [comment]
A solution here: https://medium.com/@gajus/making-the-anchor-links-work-in-spa-applications-618ba2c6954a

**Comment by xcatliu** [comment]
Fixed in v0.9.2
